### DISCUSSION TITLE
Error Invariant Violation

### DISCUSSION THREAD

**Discussion by Victorcassiano** [software-mansion/react-native-reanimated#2344] [General]
I did the step by step to install Moti, using only React Native, without Expo.
returned these errors to me:

<img src="https://i.imgur.com/a551x0W.png" alt="Minha Figura" width="400px">

<img src="https://i.imgur.com/TbOZ8yd.png" alt="Minha Figura" width="900px">

I've remade the step by step several times

**Comment by piaskowyk** [comment]
Duplicate of: https://github.com/software-mansion/react-native-reanimated/discussions/2346
